introduction.md

```diff
@@ -1,3 +1,4 @@
 * Ruby path is my favorite
 * \c-=
 * Java
+* bew
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?